backend/apps/core/models.py

```
@@ -319,3 +319,31 @@ class Lesson(TenantAwareModel):
+    db_query_count = models.IntegerField()
+    db_duration_ms = models.FloatField()
+    cache_hits = models.IntegerField(default=0)
+    cache_misses = models.IntegerField(default=0)
+    serialization_ms = models.FloatField(default=0)
+    timestamp = models.DateTimeField(auto_now_add=True, db_index=True)
+    user_id = models.IntegerField(
+        null=True, blank=True, help_text="Anonymized after 24h"
+    )
+
+    class Meta:
+        ordering = ["-timestamp"]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)